app/payments/razorpay.php

```diff
@@ -71,7 +71,14 @@
         $url = fn_url("payment_notification.return?payment=razorpay", AREA, 'current');
 
         $data = $razorpayPayment->getOrderData($order_id, $order_info, $processor_data);
-        
+
+        if ($data['currency'] === "KWD" or
```

Comment:
can we put the unsupported currencies in a constant and use the constant to check?